docs/video/01-build/01-Develop-Get-Started.mdx

```diff
@@ -69,19 +68,19 @@ The following speed run is designed to give you a quick overview of usage and de
       video: true, // to enable video on start
       audio: true, // to enable audio on start
     },
-    media?: MediaStream, // to pass your existing media stream(for backend use)
+    media?: MediaStream, // to pass your existing media stream(only for backend use)
   });
 
   // Firing a video call request to a recipient
   await aliceVideoCall.request([recipient]); // see supported wallet standards - https://push.org/docs/video/supported-wallet-standards
 
   // To listen to real-time video calls
-  Stream.on(CONSTANTS.STREAM.VIDEO, async (data) => {
+  stream.on(CONSTANTS.STREAM.VIDEO, async (data : TYPES.VIDEO.EVENT) => {
     if (data.event === CONSTANTS.STREAM.VIDEO.REQUEST) {
       // RequestVideo Event is fired when there is an incoming video call request
       // you can show an UI modal to accept or reject the call
-      // store this data.peerInfo which will be used to accept the call when calling approve() method
-      peerInfo = data.peerInfo;
+      // store the incoming peer address which will be used to accept the call when calling approve()/deny() method
+      incomingCallerAddress = data.peerInfo.address
```

Comment:
I don't think this is needed since approve also defaults to approving the first caller. you can expand on this on Handle Incoming video call section